### PR TITLE
Register sewt.is-a.dev

### PR DESCRIPTION
**Pull request by SkyExploreWasTaken**
Register sewt.is-a.dev with CNAME record pointing to SkyExploreWasTaken.github.io.

### CODE REVIEW

File: domains/sewt.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "SkyExploreWasTaken",
+           "email": "",
+           "discord": "1049263707177353247",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.JjnXMeB-5yChFwqeTjRtzK3S_0JshRprm7Ix_Yl-zlvBzrR4ZlVNljlNwz5hHd6RwuBC_y5QFA1wpuLY7J9tuvaW_ttJpnbU58UGV2JNs6yR0C6ZBp5S0XR7J0zKoloVPgoN9i4q8SOzJeBQCanVOFzX3d9haVlZxrZFInLbMybS8FegbUdzDuV6s6TB0gA8fpBWx29ftzsLfwVj96tO9gmB0cJAU-URofHxmMBDUCIXDELQdrlRl_sj4kjW3uIKus_lgN7GU55xIvErfpV2_NVZTiuPzOh5HcP7rwWHhDaeKvQL4HeKWxIlrqQhPtQHQTFTgJuh7MmJimD95ixgVg.fhYccie9reT4Mi7XAmEtZA.lNh66dK74iYl7grB--wq1yi-AEKzvkuN7TLOS75XwYxJskb4dSr8TUcM0Y5ZVfR27oMahE1_kerqZOZOtZW58ZFwqoFD0oCFaEm68nwTd3or3uBWcYbZk5ilghu8TpPN.zKRzlPiGTBfNQTbJkCbrGg"
+        },
+    
+        "record": {
+            "CNAME": "skyexplorewastaken.github.io"
```
Comment (CuteDog5695) [inline domains/sewt.json:10]: 404 not found, please give a preview of your website.

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by SkyExploreWasTaken** [comment]
Sorry I will upload it in a few minutes I just used the command on discord before

**Comment by SkyExploreWasTaken** [comment]
Oh my god sorry I confused it I thinked the Discord command will create the site with my username but it just created a register repository

**Comment by SkyExploreWasTaken** [comment]
@CuteDog5695 I merged my site it should work now

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**